### DISCUSSION TITLE
Is it possible to integrate the Austrian Map ÖK50 into OSMAnd?

### DISCUSSION THREAD

**Discussion by Wonderer8** [osmandapp/OsmAnd#18691] [Ideas]
Good day

I've been an avid user of OSMAnd for a few years now,
also registered on open streetmap in order to make changes and improvements
and would have a question about the map source:

Is it possible to integrate the Austrian Map ÖK50 into OSMAnd?

On the homepage of the provider (public office/authority)

https://portal.bev.gv.at/portal/page?_pageid=713,1570702&_dad=portal&_schema=PORTAL
I found this information

"Online access

The data of the ÖK50-UTM can be downloaded online free of charge via www.austrianmap.at
be visualized."

>>>

https://maps.bev.gv.at/#/center/13.35,47.77/zoom/6.2

In addition to https://basemap.at/, it is another publicly available map.
which, however, is more suitable for hiking, for example.

For the most part, public hiking trails are marked by default.

Thank you very much!

Yours sincerely,

**Comment by mariush444** [comment]
Depends what you mean integrate.
Online maps can be added as ovrrlay or underlay.

Problem with your maps is that they use lat lon but osmand supports: xyz, bbox, q (quadkey)

https://osmand.net/docs/technical/osmand-file-formats/osmand-metainfo/

You have to find link with those tags i stead lat lon

**Comment by mariush444** [comment]
I notice that there was a WMTS for https://basemap.at/


so based on https://mapsneu.wien.gv.at/basemapneu/1.0.0/WMTSCapabilities.xml
you can use online map creating .metainfo file 

[url_template]
https://mapsneu.wien.gv.at/basemap/bmaporthofoto30cm/normal/google3857/{$z}/{$y}/{$x}.jpeg
[ext]
.jpeg
[min_zoom]
1
[max_zoom]
20
[tile_size]
256
[img_density]
16
[avg_img_size]
18000

you can add other layers to osmand using this example and WMTS's definition

**Comment by wolfbert** [comment]
I've created an offline sqlite database from the ÖK50 GEOTIFF files. Works great, if you don't mind the storage requirements (all of Austria would be approx 6 GB). I'm happy to share the details. I have not yet experimented with the WMTS layer described above, but that might be even better.

Edit: Just noticed, the above WMTS is for basemap.at, which is different from the ÖK50.

**Reply by vshcherb** [reply]
We don't support Geotif files yet...

**Reply by mariush444** [reply]
@wolfbert 
Where is link to ÖK50 ?

**Reply by wolfbert** [reply]
To get an impression, you can view the map online: [https://maps.bev.gv.at/](https://maps.bev.gv.at/) (zoom in as far as possible)
There is also a viewer app on [Google Play](https://play.google.com/store/search?q=austrian+map+mobile&c=apps&hl=de_AT&gl=US)

The GEOTIFF files (CC-BY-4.0) for 1:50.000 can be found [here](https://data.bev.gv.at/geonetwork/srv/ger/catalog.search#/search?facet.q=keyword%2FRaster%26format%2FGeoTIFF&resultType=details&sortBy=relevance&any=km50&from=1&to=100&fast=index&_content_type=json) (switch to list view to see the file number, and switch language to English for a limited amount of translation). Each file covers 50x50 km (20.000x20.000 px). The overview for the file numbers is [here]( https://data.bev.gv.at/download/KM_R/KM50/KM50R_Kacheln_Uebersicht.pdf).

To download a specific file, proceed as follows:

1. Click on the file name
2. Open the tab "Downloads, Ansichten und Links" ("Downloads, views, and links")
3. Click on the link with the .tif extension (4th in the list) or on "Herunterladen" ("Download")

To build an sqlite database, open the file in MAPC2MAPC (with GDAL) as calibrated map and write for mobile device (Locusmap etc. format) just for level 15. Each map file adds about 100 MB.

Edit: There is also a 1:250.000 version of this map [here](https://data.bev.gv.at/geonetwork/srv/ger/catalog.search#/search?facet.q=format%2FTIFF&resultType=details&sortBy=relevance&any=km250&from=1&to=100&fast=index&_content_type=json), with [overview](https://www.bev.gv.at/dam/jcr:385c2ac2-54a0-4dde-89a6-04e71b28b9cc/KM250R-Kacheln_Uebersicht.pdf). It comes in slightly different format. There are two files (UTM32 and UTM33) and you need the "KM250-R UTM32 Farbbild" and "KM250-R UTM33 Farbbild" (which include shading) from the latest version (1.07.2023). It is obviously less useful for hiking ;-) but still a very good overview map.
